compiler/rustc_parse/src/parser/item.rs

```diff
@@ -825,6 +858,10 @@ impl<'a> Parser<'a> {
             self.expect_semi()?;
 
             let whole_span = lo.to(self.prev_token.span);
+            if !matches!(impl_restriction.level, RestrictionLevel::Implied) {
+                let msg = "trait aliases cannot be implemented";
+                self.struct_span_err(whole_span, msg).span_label(whole_span, msg).emit();
+            }
```

Comment:
1) The error message is somewhat confusing — "cannot be implemented" is meh
2) Why have a label which is just the same as the error?
3) Why isn't this translatable?

Reply:
Do you (or @estebank) have a suggestion for a better error message?